### PR TITLE
Draft CRAN release (v1.0.0)

### PR DESCRIPTION
**Pull request by lawinslow**
Manuscript and package have BAO approval. Preparing for CRAN release, fixing mostly documentation issues with check --as-cran build on latest (6-20-2016) R build. 

@anybody want to give it a quick review?

### CODE REVIEW

**Comment by aappling-usgs** [comment]
1.0.0!!

**Comment by aappling-usgs** [comment]
quick skim looks good to me

File: R/item_rm_recursive.R [hunk 1]
```diff
@@ -3,8 +3,9 @@
 #' @keywords internal
 #' @template manipulate_item
 #' @return \code{TRUE} to indicate success
-#' @details BEWARE: This completely removes ALL CHILD ITEMS AND THEIR CHILDREN
+#' @description BEWARE: This completely removes ALL CHILD ITEMS AND THEIR CHILDREN
```
Comment (jread-usgs) [inline R/item_rm_recursive.R]: a haunting message...

Comment (lawinslow) [inline R/item_rm_recursive.R]: Nobody messes with item_rm_recursive.

**Pull request merged**